pkg/doc/vc/validate.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vc
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/kms"
+)
+
+const (
+	jwtVC = "jwt_vc"
+	ldpVC = "ldp_vc"
+)
+
+/*
+"*/
+
+func ValidateVCFormat(format string) error {
+	if format != jwtVC && format != ldpVC {
+		return fmt.Errorf("unsupported vc format %s", format)
+	}
+	return nil
+}
+
+func ValidateVCSignatureAlgorithm(format, signatureType string) error {
+	var validSignatureTypes []string
+
+	if format == jwtVC {
+		validSignatureTypes = []string{"EdDSA",
+			"ES256k",
+			"ES256",
+			"ES384",
+			"PS256",
+		}
+	}
+
+	if format == ldpVC {
+		validSignatureTypes = []string{
+			"Ed25519Signature2018",
+			"Ed25519Signature2020",
+			"EcdsaSecp256k1Signature2019",
+			"BbsBlsSignature2020",
+			"JsonWebSignature2020",
```

Comment:
can we keep these in constants?
something like this https://github.com/hyperledger/aries-framework-go/blob/main/pkg/wallet/contents.go#L31-L73

And try to make checks case insensitive

Reply:
objective: should be easier to add new key types in future.